openvpn-install.sh

```diff
@@ -1286,6 +1286,35 @@ function removeOpenVPN() {
 	fi
 }
 
+function listCerts () {
+# Original Script from PiVPN: list clients script
+# Modified Script to add Certificate expiration Date -- psgoundar  
+INDEX="/etc/openvpn/easy-rsa/pki/index.txt"
+printf "\\n"
+if [ ! -f "${INDEX}" ]; then
+        echo "The file: $INDEX was not found!"
+        exit 1
+fi
+printf "\\e[1m::: Certificate Status List :::\\e[0m\\n"
+printf "\\n%6s\\t%14s\\t%22s\\n" "Status" "Name" "Expiration"
+printf '=%.0s' {1..50}
+printf '\n'
+while read -r line || [ -n "$line" ]; do
+    STATUS=$(echo "$line" | awk '{print $1}')
+    NAME=$(echo "$line" | cut -d '=' -f2)
+    EXPD=$(echo "$line" | awk '{if (length($2) == 15) print $2; else print "20"$2}' | cut -b 1-8 | date +"%b %d %Y" -f -)
```

Comment:
```suggestion
    EXPD=$(echo "$line" | awk '{if (length($3) == 13) print "20"$3; else print "20"$2}' | cut -b 1-8 | date +"%b %d %Y" -f -)
```
You just missed to check the third column first. Now it should work for revoked certs too.